doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -124,7 +124,10 @@ nRF IEEE 802.15.4 radio driver
 Wi-Fi
 -----
 
-|no_changes_yet_note|
+* Added:
+
+  * Integration of Wi-Fi connectivity with Connection Manager connectivity API.
```

Comment:
If it is that link, then please use this:

```suggestion
  * Integration of Wi-Fi connectivity with the :ref:`Connection Management <zephyr:bluetooth-connection-management>` API.
```

Reply:
Marking this resolved since there is no documentation for conn_mgr yet